skopt/tests/test_parallel_cl.py

```diff
@@ -0,0 +1,121 @@
+"""This script contains set of functions that test parallel optimization with
+skopt, where constant liar parallelization strategy is used.
+"""
+
+
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_raises
+
+from skopt.space import Real
+from skopt import Optimizer
+from skopt.benchmarks import branin
+import skopt.learning as sol
+
+from scipy.spatial.distance import pdist
+import pytest
+
+# list of all strategies for parallelization
+supported_strategies = ["cl_min", "cl_mean", "cl_max"]
+# Extract available surrogates, so that new ones are used automatically
+available_surrogates = [
+    getattr(sol, name) for name in sol.__all__
+    if "GradientBoostingQuantileRegressor" not in name
+]  # excluding the GradientBoostingQuantileRegressor, will open issue later
```

Comment:
Actually now that I think about it I believe I already opened an issue about this in #348 - somehow default acquisition function does not play well with this surrogate.